src/Middleware/RemoveComments.php

```diff
@@ -7,7 +7,8 @@ class RemoveComments extends PageSpeed
     public function apply($buffer)
     {
         $replace = [
-            '/<!--[^]><!\[](.*?)[^\]]-->/s' => ''
+            '/<!--[^]><!\[](.*?)[^\]]-->/s' => '',
+            '/(?:(?:\/\*(?:[^*]|(?:\*+[^*\/]))*\*+\/)|(?:(?<!\:|\\\|\'|\")\/\/.*))/' => '',
```

Comment:
It's removing javascript comments at html context.
I'll try to work in a solution for this problem.

Reply:
@lucasMesquitaBorges Thanks!!!

Reply:
I'm into as well. Thanks

Reply:
Hi, I am new to this, firstly thanks for this package, currently I have the same problem, I just wanna ask if the new version will get released soon or if there's a temporary solution ??

Reply:
Hi @hajar-alpha, thanks!

We are working to fix all the bugs.

Reply:
Hey @lucasMesquitaBorges, any news? 😅

Reply:
Hi @joaorobertopb I believe i'm finishing this fix. If everything goes right, this afternoon I'll push my commit fixing this issue.

Reply:
Sorry, i wasn't able to push it yesterday. This regex is a little bit more tricky that i thought it would be. When I started fixing it, i saw haw many situations this regex could break.

Those situations will be more obvious with the tests cases that i'll create.

Reply:
Hey @lucasMesquitaBorges , let's change the target these fixes to branch **2.x**. OK?